### ISSUE TITLE
JS-SDK Stored Interactions for the StakingCollection transactions 

### ISSUE THREAD

**Issue by caseyturnerarrington** [onflow/flow-core-contracts#126]
Based on the work that is done via ticket https://app.zenhub.com/workspaces/flow-5a2ad022cee40e25b0885f82/issues/dapperlabs/flow-internal/1467, we will need stored interactions that implement the transactions that were created via this ticket to ensure Flow Port can call these transactions successfully.

**Comment by caseyturnerarrington** [comment]
This is no put on hold until we get a ledger resource to unblock the remaining work needed for the ledger tx's.

**Comment by JeffreyDoyle** [comment]
Closing as this as been implemented and shipped :)
